app/routes.py

```diff
@@ -1 +1,102 @@
-from flask import Blueprint
\ No newline at end of file
+from flask import Blueprint, jsonify, abort, make_response, request
+from app.models.task import Task
+from app import db
+from sqlalchemy import asc
+from datetime import datetime
+from datetime import timezone
+import requests
+from dotenv import load_dotenv
+import os
+load_dotenv()
+
+task_bp = Blueprint("tasks", __name__, url_prefix="/tasks")
+
+def validate_model(cls, model_id):
+    try:
+        model_id=int(model_id)
+    except:
+        abort(make_response({"message":f"{cls.__name__}{model_id} invalid"}, 400))
+    model =  cls.query.get(model_id)
+    if not model:
+        abort(make_response({"message":f"{cls.__name__} {model_id} was not found"}, 404))
+    return model
+
+@task_bp.route("", methods = ["GET"])
+def read_all_tasks():
+    query_sort= request.args.get("sort")
+    if query_sort:
+        if query_sort == "asc":
+            tasks = Task.query.order_by(Task.title.asc())
+        elif query_sort == "desc":
+            tasks = Task.query.order_by(Task.title.desc())
+    else:
+        tasks = Task.query.all()
+
+    tasks_data=[task.to_dict() for task in tasks]
+    return jsonify(tasks_data)
+
+@task_bp.route("/<task_id>", methods = ["GET"])
+def read_one_task(task_id):
+
+    task = validate_model(Task, task_id)
+    
+    return {"task":task.to_dict()}
+
+@task_bp.route("", methods = ["POST"])
+def create_task():
+    try:
+        request_body = request.get_json()
+        new_task = Task.from_dict(request_body)
+    except:
+        abort(make_response({"details": "Invalid data"}, 400))
+    db.session.add(new_task)
+    db.session.commit()
+    return {"task":new_task.to_dict()}, 201
+
+@task_bp.route("/<task_id>", methods=["PUT"])
+def update_task(task_id):
+    task = validate_model(Task, task_id)
+    request_body = request.get_json()
+    task.title = request_body["title"]
+    task.description = request_body["description"]
+    db.session.commit()
+    return {"task":task.to_dict()}, 200
+
+@task_bp.route("/<task_id>", methods=["DELETE"])
+def delete_task(task_id):
+    task = validate_model(Task, task_id)
+    db.session.delete(task)
+    db.session.commit()
+    return make_response(jsonify({"details": f'Task {task.task_id} "{task.title}" successfully deleted'}),200)
+
+
+def send_task_to_slack_api(task):
+    text = f"Someone just completed the task {task.title}"
+    SLACK_API_KEY =  os.environ.get("SLACKAPI_TOKEN")
```

Comment:
I'm guessing that the value of `SLACKAPI_TOKEN` in your .env file starts with "Bearer ". That works totally fine here but what if you need to use that token elsewhere in your code without the "Bearer " prefix? An alternative would be to use an f-string to interpolate the token, stored exactly as it comes from Slack, and add the "Bearer " part, e.g. `f"Bearer xoxb-..."`